docs/news/news-v5.0.x.rst

```diff
@@ -0,0 +1,282 @@
+  for OSC support using TCP, or other providers.
+- Open MPI now links -levent_core instead of -levent.
+- common/cuda docs: Various fixes. Thanks to
+  Simon Byrne <simonbyrne@gmail.com> for finding and fixing.
+- osc/ucx: Add support for acc_single_intrinsic.
+- Fixed ``buildrpm.sh -r`` option used for RPM options specification.
+  Thanks to John K. McIver III <john.mciver.iii@gmail.com> for
+  reporting and fixing.
+- configure: Added support for setting the wrapper C compiler.
+  Adds new option ``--with-wrapper-cc=``.
+- mpi_f08: Fixed Fortran-8-byte-INTEGER vs. C-4-byte-int issue.
+  Thanks to @ahaichen for reporting the bug.
+- opal/thread/tsd: Added thread-specific-data (tsd) api.
+- Add missing ``MPI_Status`` conversion subroutines:
+  ``MPI_Status_c2f08()``, ``MPI_Status_f082c()``, ``MPI_Status_f082f()``,
+  ``MPI_Status_f2f08()`` and the ``PMPI_*`` related subroutines.
+- patcher: Removed the Linux component.
+- opal/util: Fixed typo in error string. Thanks to
+  NARIBAYASHI Akira <a.naribayashi@fujitsu.com> for finding
+  and fixing the bug.
+- Fortran/use-mpi-f08: Generate PMPI bindings from the MPI bindings.
+- Converted man pages to markdown.
+  Thanks to Fangcong Yin <fyin2@nd.edu> for their contribution
+  to this effort.
+- Fixed ompi_proc_world error string and some comments in pml/ob1.
+  Thanks to Julien EMMANUEL <julien.emmanuel@inria.fr> for
+  finding and fixing these issues.
+- oshmem/tools/oshmem_info: Fixed Fortran keyword issue when
+  compiling param.c. Thanks to Pak Lui <pak.lui@amd.com> for
+  finding and fixing the bug.
+- autogen.pl: Patched libtool.m4 for OSX Big Sur. Thanks to
+  @fxcoudert for reporting the issue.
+- Updgraded to HWLOC v2.4.0.
+- Removed config/opal_check_pmi.m4.
+  Thanks to Zach Osman <zosman@gmu.edu> for the contribution.
+- opal/atomics: Added load-linked, store-conditional atomics for
+  AArch6.
+- Fixed envvar names to OMPI_MCA_orte_precondition_transports.
+  Thanks to Marisa Roman <marisa.roman@cornelisnetworks.com>
+  for the contribution.
+- fcoll/two_phase: Removed the component. All scenerios it was
+  used for has been replaced.
+- btl/uct: Bumped highest supported UCX version to v1.10.x.
+- btl/uct: Updated the defaults to allow Mellanox HCAs
+  (mlx4_0, and mlx5_0) for osc/rdma to work out-of-the-box.
+- ULFM Fault Tolerance has been added. See ``README.FT.ULFM.md``.
+- Fixed a crash during CUDA initialization.
+  Thanks to Yaz Saito <yasushi.saito@gmail.com> for finding
+  and fixing the bug.
+- Added CUDA support to the OFI MTL.
+- ompio: Added atomicity support.
+- Singleton comm spawn support has been fixed.
+- Autoconf v2.7 support has been updated.
+- Fortran: Added check for ``ISO_FORTRAN_ENV:REAL16``. Thanks to
+  Jeff Hammond <jeff_hammond@acm.org> for reporting this issue.
+- Changed the MCA component build style default to static.
+- PowerPC atomics: Force usage of opal/ppc assembly.
+- Removed C++ compiler requirement to build Open MPI.
```

Comment:
Are you 100% sure that this is true?

Reply:
https://github.com/open-mpi/ompi/pull/9075

it's my understanding of that change.